test/functionalTest/cases/4286_update_format_for_notification_messages/update_format_for_notification_messages_custom_ngsi_patching.test

```diff
@@ -0,0 +1,211 @@
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+echo "02. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedKeyValues with httpCustom and ngsi field adding a new attribute"
+echo "=========================================================================================================================================="
+payload='
+{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1"
+      }
+    ],
+    "condition": {
+      "attrs": [ "A1" ]
+    }
+  },
+  "notification": {
+    "httpCustom": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify",
+      "ngsi": {
+        "id": "E1",
+        "type": "T1",
+        "A3": {
+          "value": 1,
+          "type": "Number"
+        }
+      }
+    },
+    "attrs": [ "A1" ],
+    "attrsFormat": "simplifiedKeyValues"
+  },
+  "expires": "2050-04-05T14:00:00.00Z"
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+echo "03. Create entity E1/T1"
+echo "======================="
+payload='{
+  "id": "E1",
+  "type": "T1",
+  "A1": {
+    "value": 1,
+    "type": "Number"
+  }
+}'
+orionCurl --url /v2/entities --payload "${payload}"
+echo
+echo
+
+
+echo "04. Dump accumulator to see notifications"
+echo "========================================="
+accumulatorDump
+echo
+echo
+
+
+--REGEXPECT--
+01. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedNormalized with httpCustom and ngsi field adding a new attribute
+===========================================================================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedKeyValues with httpCustom and ngsi field adding a new attribute
+==========================================================================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+03. Create entity E1/T1
+=======================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E1?type=T1
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+04. Dump accumulator to see notifications
+#SORT_START
+=========================================
+POST http://localhost:REGEX(\d+)/notify
+Fiware-Servicepath: /
+Content-Length: 70
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: simplifiedNormalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36}; cbnotif=[12345])
+
+{
+    "A1": {
+        "metadata": {},
+        "type": "Number",
+        "value": 1
+    },
+    "id": "E1",
+    "type": "T1"
+}
+#SORT_END
+#SORT_START
```

Comment:
```suggestion
```

Both notification should be within the same SORT_START - SORT_END block, to tell the test framework that the order between them doesn't matter.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4298/commits/7bf26136d4ee0b7fa8abd10773fcf7ef1e3e043a